cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/web/OAuth20RevokeClientTokensController.java

```diff
@@ -68,11 +73,29 @@ protected ModelAndView handleRequestInternal(final HttpServletRequest request, f
             return OAuthUtils.writeJsonError(response, OAuthConstants.INVALID_REQUEST, e.getMessage(), HttpStatus.SC_BAD_REQUEST);
         }
 
-        if (!centralOAuthService.revokeClientTokens(clientId, clientSecret)) {
```

Comment:
The functionality of `revokeClientTokens()` are removed and directly handled by the controller here in `OAuth20RevokeClientTokensController. handleRequestInternal()`. Please refer to the code below and the three following comments